### DISCUSSION TITLE
Logging function name and line number

### DISCUSSION THREAD

**Discussion by lewysigns** [anvilistas/anvil-extras#332] [Q&A] [answered]
Good Morning/Afternoon (Pending Location!) 

I was trying to mimic the function and line number formatting in the logging library here: [https://stackoverflow.com/questions/10973362/python-logging-function-name-file-name-line-number-using-a-single-file](url)

Is this functionality apart of the logging module? I did not see it in the documentation.

Here is my logger that works:
```
app_logger  = Logger(name="app_log",level=DEBUG,format="{name}-{level} {datetime:%Y-%m-%d %H:%M:%S}: {msg}")
```

And here my attempt that breaks:
```
app_logger  = Logger(name="app_log",level=DEBUG,format="{name}-{level}-{funcName} {datetime:%Y-%m-%d %H:%M:%S}: {msg}")
```

Thanks!

**Comment by s-cork** [comment, marked as answer]
That's not supported i'm afraid.
The anvil-extras logger is very minimal, it doesn't know which function it is contained within when the logging happens.

**Reply by lewysigns** [reply]
Thanks for the quick answer!
